cirq/google/api/v2/sweeps.py

```diff
@@ -54,6 +54,17 @@ def sweep_to_proto(
         out.single_sweep.parameter_key = sweep.key
         for point in sweep.points:
             out.single_sweep.points.points.append(point)
```

Comment:
This isn't part of the PR but I noticed this.  It should use `extend` instead of a loop.

Reply:
Done.